libs/net/entities/Content.cs

```diff
@@ -359,7 +359,7 @@ public Content(string uid, string headline, Source source, ContentType contentTy
     public Content(string uid, string headline, string otherSource, Source source, ContentType contentType, License license, Product product, User? owner = null)
         : this(uid, headline, otherSource, contentType, license, product, owner)
     {
-        this.SourceId = source?.Id ?? throw new ArgumentNullException(nameof(source));
+        if (source != null) this.SourceId = source.Id;
```

Comment:
What is driving this modification?

Reply:
Good question, currently we are using the version 7.17.5 of Elasticsearch.Net instead of the latest version 8 (which has limited functionalities), when initiating the ElasticClient for some reasons the source is null (same for a few other places as well) which raise exception here, it seems for version 8 there is no such issue though, however due to version 8's current limited functionalities, we could not use it yet, so this would be the reason for driving this modification. Thanks

Reply:
Then we'll need to create models for Elasticsearch and not pollute the entities for the database if there is no way to maintain valid data.

Why is Elasticsearch using these constructors anyway?  As far as I can tell it should simply be deserializing which would use the default constructor without any parameters.

Reply:
After removing one of the unused constructor and changing the order of the other constructors, rollbacking these changes seem to be working fine now, but I'm not quite sure why this could make a difference, and I'm not sure why Elasticsearch would use these constructors yet.